### PR TITLE
Multiple V0 Modifications by Maverick_

### PR DESCRIPTION
**Pull request by Maverick-3D**
* [x ] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x ] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x ] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x ] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x ] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x ] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x ] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x ] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x ] Submitted STL files are oriented and scaled properly for printing.
  * [x ] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
This PR contains a number of mods that I designed and used in my V0 build. This includes:

1.  **C14 Inlet Foot** - Modification of the rear left foot to accommodate a socket for powering the V0.
2. **Deck Logo LED** - A purely cosmetic addition that places the Voron logo on the bottom deck with a Neopixel ring.
3. **LED Front Bed Mount** - Allows for the placement of a Neopixel Jewel in the face of the front bed mount.
4. **LED Hinge** - Similar to the above, but in the top right bracket of the door.
5. **Pi Cam Mount** - A simple solution for mounting a Pi Cam on a corner bracket of the TopHat.
6. **TopHat Mini 12864** - Modified the V2 case and the V0 TopHat to be used in mounting a Mini 12864 display.

#### How was it tested? 
Through numerous trial and error attempts during the process of building my V0. All of these mods are currently in use on my printer and most have been shared with other V0 builders.
#### Any background context you want to provide?
This is my first time doing a PR and I've just recently become somewhat familiar with GitHub, so it's likely that I did something incorrectly. Please let me know if anything needs to be changed.
#### Screenshots (if appropriate)
Images located in each respective folder.
#### Further notes
None

### CODE REVIEW

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mods.  Everything looks good, but can you change the URLs for images and links from absolute to relative?

Checked:

- mods in correct directories
- no spaces in directory names
- there's a readme for each mod
- top level readme is updated
- STL are oriented correctly and don't require supports

File: printer_mods/README.md [hunk 1]
```diff
@@ -63,6 +63,12 @@ like so:
 |madros |[V0 OLED-skirt](./madros/V0_skirt_OLED) |V0 skirt with 0.96" OLED|:heavy_check_mark: |:x: |:x: |
 |MarcPot |[UHP PSU Bracket Clip](./MarcPot/UHP_PSU_Bracket_Clip)|DIN rail mount for the Mean Well UHP series PSU. |:grey_question: |:grey_question: |:heavy_check_mark: |
 | |[Hour Counter Skirt + Keystone Skirt](./MarcPot/Skirt_Mods)|Skirt mod for a mechanical hour counter and keystone inserts |:x: |:x: |:heavy_check_mark: |
+|Maverick_ | [V0 C14 Inlet](https://github.com/Maverick-3D/VoronUsers/tree/master/printer_mods/Maverick_/V0_C14_Inlet_Foot) | A modification to the rear foot of the V0 to allow for the addition of a C14 Inlet. | :heavy_check_mark: | :x: | :x: |
```
[the remark this answers is not in the document]
Comment (Maverick-3D) [reply printer_mods/README.md:66]: Sure thing! I was wondering about that. I'll get right on it.

File: printer_mods/Maverick_/V0_C14_Inlet_Foot/ReadMe.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# V0 C14 Inlet
+
+A simple modification to the rear foot of the V0 to allow for the addition of a C14 Inlet.
+
+*Note - this is not compatible with the rear, lower deck panel that sits below the Z Mount.*
+
+![C14 Inlet](https://github.com/Maverick-3D/VoronUsers/blob/master/printer_mods/Maverick_/V0_C14_Inlet_Foot/Images/C14_Inlet.PNG)
```
[the remark this answers is not in the document]
Comment (Maverick-3D) [reply printer_mods/Maverick_/V0_C14_Inlet_Foot/ReadMe.md]: Got it! Should have fixed everything.

File: printer_mods/Maverick_/V0_C14_Inlet_Foot/Images/ReadMe.md [hunk 3]
```diff
@@ -0,0 +1 @@
+![C14 Inlet](https://github.com/Maverick-3D/VoronUsers/blob/master/printer_mods/Maverick_/V0_C14_Inlet_Foot/Images/C14_Inlet.PNG)
```
[the remark this answers is not in the document]
Comment (Maverick-3D) [reply printer_mods/Maverick_/V0_C14_Inlet_Foot/Images/ReadMe.md]: I just removed all of the readme files in the images folders. Should be good to go now.

**Review by KiloQubit — APPROVED** [review]
Looks good!

**Review by FHeilmann — CHANGES_REQUESTED** [review]
Hi, thanks for your submissions. 

There's currently another mod like your pi cam mod submitted in PR #177 so this mod would be a duplicate for that mod (the PR is currently under review because of another included mod, the pi cam mod will likely be merged).

Are there any significant differences between that mod and yours?

Separately, Voron printers don't need any glue and I'd like to try and keep it that way. Is there a possibility for you to modify the parts so they friction-fit (or even better, snap fit) together?

**Comment by Maverick-3D** [comment]
Hey @FHeilmann, thanks for the review and feedback. 

I looked at PR #177 and you're correct, it is similar in function. I'll remove mine from this PR.

Would changing the description from "super glue" to VHB tape be sufficient? If not, I have a variation that I can uploaded in its place.

**Comment by FHeilmann** [comment]
> Would changing the description from "super glue" to VHB tape be sufficient? If not, I have a variation that I can uploaded in its place.
Will that retain the transparency properties of your mod? If so then yes, using VHB tape is acceptable although if you had something like a snap or friction fit that would be preferable.

bests,
-Florian

**Comment by Maverick-3D** [comment]
Alright, I removed the Pi Cam mount and uploaded a different version of the deck panel that snaps together. I also revised the instructions associated with these changes. 

Please let me know if there's anything else that needs to be corrected.

**Comment by FHeilmann** [comment]
Hi, it seems like these changes did not make it into this PR. At least I can still see the references to superglue. Did you push your changes?

-Florian

**Comment by Maverick-3D** [comment]
So sorry, it didn't push. Should be good now.

**Pull request merged**